samples/meetings-visual-indicator/nodejs/pages/hello.html

```diff
@@ -0,0 +1,4 @@
+
+<div class="font-semibold font-title" style="color:gray">"Hello, Welcome to Microsoft Teams"</div>
+
+<h3 style="color:dodgerblue;font-family:sans-serif">Visual Indicator Notifications In Meetings!</h3>
```

Comment:
Visual Indicator in Teams meetings notification

Reply:
fixed